portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -50,24 +50,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
 
-    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
why is this being deleted?

Reply:
I deleted it because that is the next 5th step of week 3, and I want to get the step 4 code reviewed first.